integration-cli/docker_cli_run_test.go

```diff
@@ -1922,8 +1925,8 @@ func (s *DockerSuite) TestRunBindMounts(c *check.C) {
 	defer os.RemoveAll(tmpDir)
 	writeFile(path.Join(tmpDir, "touch-me"), "", c)
 
-	// TODO Windows Post TP5. Windows does not yet support :ro binds
-	if daemonPlatform != "windows" {
+	// TODO Windows: Temporary check - remove once TP5 support is dropped
+	if daemonPlatform == "windows" && windowsDaemonKV >= 14350 {
```

Comment:
I think this should be 
`if daemonPlatform != "windows" || windowsDaemonKV >= 14350 {`